petals-esb-deployer-api/src/main/java/org/ow2/petals/deployer/utils/XmlModelBuilder.java

```diff
@@ -56,7 +59,10 @@
             MARSHALLER = jaxbContext.createMarshaller();
             MARSHALLER.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, true);
             UNMARSHALLER = jaxbContext.createUnmarshaller();
-        } catch (JAXBException e) {
+            SchemaFactory schemaFactory = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI);
```

Comment:
I think these 3 lines are unneeded if you initialize the jaxb context with the right ObjectFactory class

Reply:
They are used to enable XML schema validation.

Reply:
ok